test/unit/app/repository/claim-repository.test.js

```diff
@@ -716,12 +716,19 @@ describe('Claim repository test', () => {
       { searchText: '113494460', searchType: 'sbi', sort: { field: 'sbi', direction: undefined } },
       { searchText: '113494460', searchType: 'sbi', sort: { field: 'sbi', direction: 'DECS' } },
       { searchText: '113494460', searchType: 'sbi', sort: undefined },
+      { searchText: 'AHWR-TEST-TEST', searchType: 'appRef' },
+      { searchType: 'adsdf' },
       { searchText: 'dfdf', searchType: 'adsdf', sort: undefined }
     ])('Search claim by search text $searchText, search type $searchType ', async ({ searchText, searchType, sort }) => {
       const callTimes = searchType !== 'adsdf' ? 1 : 0
       when(buildData.models.claim.count).mockResolvedValue(2)
       when(buildData.models.claim.findAll).mockResolvedValue(['claims1', 'claims2'])
-      await searchClaims(searchText, searchType, undefined, undefined, sort)
+      const search = {
+        text: searchText,
+        type: searchType
+      }
+      const filter = searchType === 'status' ? {} : undefined
```

Comment:
Might be worth adding a test case where the filter is actually set to something with a value other than just empty object or undefined I think, so there's something indicating the query passed through to function is as expected

Reply:
https://github.com/DEFRA/ffc-ahwr-application/pull/293/commits/d95b2239020615af7c5bb1807585d261fd5f48d6